### ISSUE TITLE
Dual White v6 commands not working

### ISSUE THREAD

**Issue by RobinBol** [mwittig/node-milight-promise#22]
I have a Milight 6W DUAL WHITE LED Lamp E27 linked to a v6 bridge, but when using the white commands (for example the example-dual-white-v6.js script) the bulb does not react, when using the new mobile app (specifically the 10th remote in the remote selector), it does work. Could it be that the commands are not correct? Anyone else having issues with the dual white bulbs? RGBW is working fine by the way!

**Comment by mwittig** [comment]
Works for me, but maybe I have an older version of the dual white bulb. Mine is working with controller 11 and 12 of the Milight iPhone app. 

![image](https://user-images.githubusercontent.com/6117742/27308655-16392412-5550-11e7-9e1b-de51a5283641.png)

![image](https://user-images.githubusercontent.com/6117742/27308673-2a0852c4-5550-11e7-923e-f689aaab32b6.png)

**Comment by mwittig** [comment]
@RobinBol any news on this? Maybe you can describe which bulb model you have and post a screenshot of the app control you're using. Thanks

**Comment by RobinBol** [comment]
Sorry for my slow response, I was not able to test some things that I wanted to test before reporting back.

So it turns out, I paired the exact same bulb you show (according to the text on the bulb) but with the following remote:

![image](https://user-images.githubusercontent.com/5762579/27432220-915a3022-574f-11e7-88c7-65932b9ef581.png)

So my guess was that this is causing the bulb to not work with your library. So I repaired the bulb with the remote you showed and it all worked. But then to be sure I repaired the bulb again with the non-functioning remote; and again it worked. So I am not sure what the problem was, but it seems the commands are correct.

I will close this issue for now, and report back if this issue keeps occuring, thanks!
